securedrop/source_templates/index.html

```diff
@@ -32,11 +32,12 @@ <h2 id="security-level-heading" hidden>{{ gettext('How to change your security l
       <li>
         {{ gettext('Click the <img src="{icon}" alt="&quot;Security Level&quot; button"> in the toolbar above').format(icon=url_for("static", filename="i/font-awesome/black/guard.svg"))  }}
       </li>
-      <li>{{ gettext('Click <b>Change</b> to open Security Level preferences') }}</li>
-      <li>{{ gettext('Select <b>Safest</b> and close the preferences tab') }}</li>
+      <li>{{ gettext('Click <b>Settings</b> to open Security Level preferences') }}</li>
+      <li>{{ gettext('Click <b>Change...</b> and select <b>Safest</b>') }}</li>
+      <li>{{ gettext('Click <b>Save and restart.</b>') }}</li>
     </ol>
     <p>
-      {{ gettext('<a href="/" aria-label="Follow these instructions, then refresh this page">Refresh this page</a>, and you\'re done!') }}
+    {{ gettext('Tor Browser will restart, with its security level set to <b>Safest</b>. After the restart, navigate back to this page.') }}
```

Comment:
Nit: I'd favor the simplest language possible, e.g.:

```suggestion
    {{ gettext('Tor Browser will restart, with its security level set to <b>Safest</b>. After the restart, come back to this site.') }}
```

I wish we could do: `...at <kbd>abcdef1234.onion</kbd>`, but the Source Interface doesn't know if it has a short onion name.

The next best thing would be: `...at the same <kbd>.onion</kbd> address you entered to get here`, but that's probably overkill.

So, whatever verb we wind up with here, thanks for finding a way to thread the usability needle!